release/sil/sil.bcc-Arab.upp_ptwl1/README.md

```diff
@@ -0,0 +1,29 @@
+Balochi lexical model
+===================
+
+© 2020 SIL International
+
+Version 1.0
+
+Description
+-----------
+
+Balochi generated from template
+
+Links
+-----
+
+Supported Platforms
+-------------------
+ * Windows
```

Comment:
Current, only the mobile platforms support lexical models. Maybe the desktop platforms came from the Developer template? But they should be removed.